README.md

```diff
@@ -46,6 +46,16 @@ module.exports = {
 };
 ```
 
+For [vitest](https://vitest.dev/) test suites:
+
+1. Create an `.eslintrc.js` file in your project root and extend the Svelte-specific configuration:
```

Comment:
- "Svelte-specific configuration:" I guess it should be adapter?

- Is that correct? A consumer probably don't want to use the vitest configuration as global eslint configuration but solely use it for test no?